extensions/interactions/NumericInput/NumericInput.js

```diff
@@ -24,21 +24,36 @@ oppia.directive('oppiaInteractiveNumericInput', [
   'oppiaHtmlEscaper', function(oppiaHtmlEscaper) {
     return {
       restrict: 'E',
-      scope: {},
+      scope: {
+        getAnswerClassificationData: '&answerClassificationData'
+      },
       templateUrl: 'interaction/NumericInput',
-      controller: ['$scope', '$attrs', 'focusService',
-          function($scope, $attrs, focusService) {
+      controller: ['$scope', '$attrs', 'focusService', 'numericInputService',
+          function($scope, $attrs, focusService, numericInputService) {
         $scope.answer = '';
         $scope.labelForFocusTarget = $attrs.labelForFocusTarget || null;
+        var answerClassificationData = $scope.getAnswerClassificationData();
 
         $scope.NUMERIC_INPUT_FORM_SCHEMA = {
           type: 'float',
           ui_config: {}
         };
 
+        var USE_FRONTEND_RULE_EVALUATION = true;
```

Comment:
I think, for now, this is supposed to be a per-interaction thing -- we will have an intermediate state of affairs where some interactions have frontend eval and others have backend eval.